### ISSUE TITLE
Support Additional Languages

### ISSUE THREAD

**Issue by ann-kilzer** [WWCodeTokyo/speak-her-db#88]
**Is your feature request related to a problem? Please describe.**
A [twitter user requested Spanish support](https://twitter.com/Karlagollanes/status/1300446160734187521?s=20) and is willing to help with translations. We could support additional languages, and make it easier for people to add new translations. 

**Describe the solution you'd like**
- Support additional spoken languages in the Nomination form as fields
- Support additional locales for the site
  - update the locale switcher to be a selector rather than a toggle
  - decide which languages to focus on first -- Should it be translator driven? Should we look at popularity? See https://en.wikipedia.org/wiki/Languages_of_Japan
      - Chinese
      - Korean
      - Portuguese
      - Hindi
      - Vietnamese
      - Thai
      - others??

**Comment by alangdm** [comment]
Hi there, I love the work you're doing with this project 💯 

As a native Spanish speaker myself, I would also love to see more language representation even if just in the spoken languages section

However, the truth is that, at least in my experience in tech events in Japan, most events only want talks in English or Japanese, so adding this support might bloat the spec in a maybe not-so-desirable way 😞 

So, I think you should all carefully consider how valuable adding this is to the project scope as it's quite a big feature

Again, thanks a lot for the work you're doing with this 👍

**Comment by alangdm** [comment]
Yeah, the infrastructure is exactly what I was concerned about, getting that right can take quite a bit of time or money 😵 💸 

However, if you do get it up running, I would be willing to collaborate with Spanish translations so ping me if you ever happen to do it 👍 

I do think that maybe allowing for at least more variety in the spoken languages section could be great, better even if you could use those as a search filter so that this database can be helpful even for people running international online events

**Comment by ann-kilzer** [comment]
Added additional languages to Airtable Languages table last night.

I selected languages based on [which nationalities are most represented in Japan](https://en.wikipedia.org/wiki/Demographics_of_Japan#Languages), then added the top languages spoken in those areas. We can always add more upon requests, and the form allows custom entries, though at least one use had difficulty with the current UX.


Here's a [helpful guide for listing languages in their native language](https://www.omniglot.com/language/names.htm). I cross-referenced with what Google uses (checking under chrome://settings, searching for Languages, then examining the search terms)
